### PR TITLE
PHP 8.3 dictionary

### PR DESCRIPTION
**Pull request by lptn**
Following methods added to PHP 8.3 but not added to the dictionary because we currently don't have a mechanism php-ext versioning
 - `\Random\Randomizer::getBytesFromString`
 - `\Random\Randomizer::getFloat()`
 - `\Random\Randomizer::nextFloat()`

Based on https://php.watch/versions/8.3 and php-src discovery

______

## Release plans

According to the [preparation tasks list](https://wiki.php.net/todo/php83), PHP 8.3 will be released on November 23, 2023, after three alpha releases, three beta, and six release candidates.

Date | Release
-- | --
June 8, 2023 | Alpha 1
June 22, 2023 | Alpha 2
July 6, 2023 | Alpha 3
July 18, 2023 | Feature freeze
July 20, 2023 | Beta 1
August 03, 2023 | Beta 2
August 17, 2023 | Beta 3
August 31, 2023 | RC 1
September 14, 2023 | RC 2
September 28, 2023 | RC 3
October 12, 2023 | RC 4
October 26, 2023 | RC 5
November 9, 2023 | RC 6
November 23, 2023 | GA

### CODE REVIEW

**Comment by orklah** [comment]
Note: InternalCallMapHandler::PHP_MINOR_VERSION should change with this PR. Callmap line 5 should too

File: dictionaries/CallMap.php [hunk 1]
```diff
@@ -6162,6 +6162,7 @@
 'json_encode' => ['non-empty-string|false', 'value'=>'mixed', 'flags='=>'int', 'depth='=>'int'],
 'json_last_error' => ['int'],
 'json_last_error_msg' => ['string'],
+'json_validate' => ['bool', 'json'=>'string', 'depth?'=>'positive-int', 'flags='=>'int'],
```
Comment (othercorey) [inline dictionaries/CallMap.php:6165]:
```suggestion
'json_validate' => ['bool', 'json'=>'string', 'depth='=>'positive-int', 'flags='=>'int'],
```

File: dictionaries/CallMap_83_delta.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ *     representing the function as it was in PHP 8.2 and in PHP 8.3, respectively
+ *
+ * @see CallMap.php
+ * @see https://php.watch/versions/8.3
+ *
+ * @phan-file-suppress PhanPluginMixedKeyNoKey (read by Phan when analyzing this file)
+ */
+return [
+  'added' => [
+    'json_validate' => ['bool', 'json'=>'string', 'depth?'=>'positive-int', 'flags='=>'int'],
```
Comment (othercorey) [inline dictionaries/CallMap_83_delta.php:20]:
```suggestion
    'json_validate' => ['bool', 'json'=>'string', 'depth='=>'positive-int', 'flags='=>'int'],
```

**Review by weirdan — APPROVED** [review]

**Comment by weirdan** [comment]
Thanks!

**Pull request merged**